edvart/report.py

```diff
@@ -225,7 +226,14 @@ def _export_html(
 
         html_exporter = nbconvert.HTMLExporter(**html_exp_kwargs)
 
+        # Workaround for a warning from `nbconvert` regarding debugging
+        # and frozen modules. We are not debugging, so we can safely ignore it.
+        disable_validation_env_var_name = "PYDEVD_DISABLE_FILE_VALIDATION"
+        env_original = os.environ.copy()
+        os.environ[disable_validation_env_var_name] = "1"
+
         html = html_exporter.from_notebook_node(nb)[0]
+        os.environ = env_original
```

Comment:
nitpick: It'd be better to implement it as a context manager so that all the code that sets/unsets the variable is contained within one uninterrupted block.

Something like:
```python
@contextmanager
def env_var(name: str, value: Any) -> None:
    original_value = os.environ[name]
    os.environ[name] = value
    try:
        yield
    finally:
        os.environ[name] = original_value
```

Reply:
Good point, added a function, with two changes:
1. `value: str` since `os.environ` values have to be strings.
2. Copying and restoring the whole environment. The environment variable being set might not be originally set, which would raise a `KeyError`.